src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -1304,6 +1321,9 @@ CLParser::VerifyOptions()
               << " use default value if it is measuring on sequence model."
               << std::endl;
   }
+  if (params_->sequence_length_variation < 0.0) {
```

Comment:
Over 100% is allowed. It's always at least 1 because of this:

https://github.com/triton-inference-server/client/blob/matthewkotila-sequence-length/src/c%2B%2B/perf_analyzer/sequence_manager.cc#L168-L170